gameSource/ExistingAccountPage.cpp

```diff
@@ -497,6 +503,21 @@ void ExistingAccountPage::actionPerformed( GUIComponent *inTarget ) {
             }
         delete [] url;
         }
+    else if( inTarget == &mOholCurseButton ) {     
+        const char *url = "https://oholcurse.com/redirect/profile";
+
+        const char *leaderboardName = getLeaderboardName();
+        unsigned char *leaderboardNameUnsigned = (unsigned char *)leaderboardName;
+        char *encodedLeaderboardName = hexEncode(leaderboardNameUnsigned, strlen((const char*)leaderboardNameUnsigned));
```

Comment:
Not a problem, but this could be simplified to:

```cpp
const char *leaderboardName = getLeaderboardName();
char *encodedLeaderboardName = hexEncode((unsigned char *)leaderboardName, strlen(leaderboardName));
```